cbor/src/main/java/com/fasterxml/jackson/dataformat/cbor/CBORParser.java

```diff
@@ -813,6 +813,9 @@ public JsonToken nextToken() throws IOException
                 return _eofAsNextToken();
             }
         }
+        if (_inputPtr < 0) {
```

Comment:
I don't think this is a very useful check: we must fix incorrect input pointer update where it occurs -- it is possible that negative offset was used to move back within buffer, but without becoming negative.